skipper.go

```diff
@@ -1580,8 +1581,12 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 			o.ClusterRatelimitMaxGroupShards = 1
 		}
 
+		failureModeFilter := ratelimitfilters.NewFailureMode()
+		failureModeFilterPostProcessor, _ = failureModeFilter.(*ratelimitfilters.FailureModeSpec)
```

Comment:
I think it is cleaner to have a separate constructor `ratelimitfilters.NewFailureModePostProcessor()` like it is done for other post-pocessors

Reply:
done